core/examples/minimal_example.py

```diff
@@ -19,16 +19,30 @@
 |Content Cell  | `Content Cell` | Content      |
 |Content Cell  | Content Cell   | Content      |
 
-
 ---
 
     #!python
     def hello():
         print('Hello World')
+---
+If you want to use math formulas, you can do this by escaping them with
+double dollar signs. To prevent build failing because of backslashes, use a
+rawstring format by adding `r` in front of the string.
+
+You can read more about the input format
+[here](http://docs.mathjax.org/en/latest/tex.html#).
 
+Example:
+
+`formula = r'$$x_{1,2} = \frac{-b \pm \sqrt{b^2-4ac}}{2b}.$$'`
+
+Renders out to this:
 """)
```

Comment:
Suggestion for example showing different ways of entering equations, labelling and corresponding equation reference, in-line symbol/equations and multi-line equations:

```
If you want to use math formulas, you have several different options. You can
use double dollar signs,

$$ \left(\frac{\sqrt x}{y^3}\right), $$

or you can wrap it inbetween special commands like this:

\begin{equation}
\cos (2\theta) = \cos^2 \theta - \sin^2 \theta \label{my_cos_equation}.
\end{equation}

All equations with labels can easily be referred to in the text as
\eqref{my_cos_equation}. If you want an equation without numbering add "notag":

\begin{equation}
\lim_{x \to \infty} \exp(-x) = 0.\notag
\end{equation}

If you want to write multi-line equations aligned on e.g. the equal sign, you 
can also do that:

\begin{align}
f(x) &= (x+a)(x+b) \\\\
     &= x^2 + (a+b)x + ab
\end{align}

The & operator indicates what to align on. You can also write in-line equations
or symbols inbetween, like \\( \alpha \\) and \\( \gamma \\).

To prevent build failing because of backslashes, use a rawstring format by adding `r` in front of the string.

You can read more about the input format [here](http://docs.mathjax.org/en/latest/tex.html#).

Example:

`formula = r'$$x_{1,2} = \frac{-b \pm \sqrt{b^2-4ac}}{2b}.$$'`

Renders out to this:
```
I had to put two backslashes when they were followed by \, ( or ), compared with MathJax documentation, to get it to work. Might have something to do with the markdown processing before it is given to MathJax.